app/migrations/Version20180619100109.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+final class Version20180619100109 extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE mooc_elements ADD share_twitter_text VARCHAR(255) NOT NULL, ADD share_facebook_text VARCHAR(255) NOT NULL, ADD share_email_subject VARCHAR(255) NOT NULL, ADD share_email_body VARCHAR(500) NOT NULL, DROP twitter_text, DROP facebook_text, DROP email_object, DROP email_body');
```

Comment:
do you have some data in dropped columns? if yes, maybe use CHANGE action instead of `ADD & DROP`?